Fix ESLint errors  during build

These errors occur while running `gatsby develop`.
```
.src/components/atoms/TextLink/index.js
  5:35  error  'children' is missing in props validation         react/prop-types
  5:45  error  'to' is missing in props validation               react/prop-types
  5:49  error  'activeClassName' is missing in props validation  react/prop-types
  5:66  error  'partiallyActive' is missing in props validation  react/prop-types
```

--- a/src/components/atoms/TextLink/index.js
+++ b/src/components/atoms/TextLink/index.js
@@ -26,7 +26,7 @@ export default function TextLink({children, to, activeClassName, partiallyActive
   }
 }
 
-Link.propTypes = {
+TextLink.propTypes = {
   children: PropTypes.node.isRequired,
   partiallyActive: PropTypes.bool,
   to: PropTypes.string.isRequired,

--- a/src/components/atoms/TextLink/index.js
+++ b/src/components/atoms/TextLink/index.js
@@ -27,6 +27,7 @@ export default function TextLink({children, to, activeClassName, partiallyActive
 }
 
 TextLink.propTypes = {
+  activeClassName: PropTypes.string,
   children: PropTypes.node.isRequired,
   partiallyActive: PropTypes.bool,
   to: PropTypes.string.isRequired,

